README.md

```diff
@@ -154,19 +154,19 @@ Internet-DrafGlue In DNS Referral Responses Is Not Optional    July 2021
    protocol extensions, when used, are also not optional.  This includes
    TSIG [RFC2845], OPT [RFC6891], and SIG(0) [RFC2931].
 
-   Glue records are added to the parent zone as part of the delegation
-   process.  They are expected to be returned as part of a referral and
-   if they can't fit in a UDP response TC=1 MUST be set to signal to the
-   client to retry over TCP.  This document reinforces that expectation.
-
-
-
-
-
+   %%% third time the bulk of this paragraph is repated Glue records are
```

Comment:
Was this supposed to be a comment? It appears in the rendered txt document, which I assume we don't want. Perhaps we just take out the repetitive text or this entire paragraph?

Reply:
So, the one new line here is "Glue records are added to the parent zone as part of the delegation process for a domain whose nameservers are contained within the domain itself.". I'll accept this PR, delete this paragraph, and add this 1 line into the Intro section to make things less repetitive.